libs/utils/energy.py

```diff
@@ -234,13 +234,7 @@ def __init__(self, target, conf, res_dir):
         self._log.info('   %s', str(self._aep.active_channels))
         self._log.debug('Results dir: %s', self._res_dir)
 
-        if self._aep is None:
-            return
-
     def reset(self):
-        if self._aep is None:
-            return
-
```

Comment:
Because it isn't related to that patch, except that it edits the same code. The patch to remove `None` checks could be merged independently.